mobingi/sesha3/sesha3_test.go

```diff
@@ -0,0 +1,12 @@
+package sesha3
+
+import (
+	"testing"
+)
+
+func TestNew(t *testing.T) {
+	url := &SeshaClientInput{URL: "https://sesha3.labs.mobingi.com:8568/d3aiwuxow4mxnsgc4j7usvcpw0bjh27kg94c/"}
+	sesha3cli, _ := New(url)
+	err := sesha3cli.Run()
```

Comment:
@biosugar0 This will wait for timeout, right? I think this will cause CI to wait for a while, which is not good. Could you try to make `Run()` to exit immediately?